src/Couchbase.Lite.Shared/Serialization/MCollection.cs

```diff
@@ -116,5 +117,14 @@ protected void Mutate()
         public abstract void FLEncode(FLEncoder* enc);
 
         #endregion
+
+        #region IDisposable
+
+        public virtual void Dispose()
+        {
+            Context?.Dispose();
```

Comment:
This makes me nervous because I don't remember who is responsible for managing this, but I don't think it is this class (what happens if two MCollections are made from the same context?)